openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/core/CoreLandmarkStorage.java

```diff
@@ -901,6 +893,17 @@ public void initLandmarkWeights(final int lmIdx, int lmNodeId, final long rowSiz
                     .append(weighting.getName()).append("|maximum=").append(finalMaxWeight.getValue() * 1.2).toString());
             }
         }
+
+        @Override
+        public void initFrom(int from, double weight){
+            super.initFrom(from, weight);
+        }
+
+        @Override
+        public void initTo(int to, double weight){
+            super.initTo(to, weight);
+        }
+
     }
```

Comment:
Do we actually need to redefine the method in the subclass if it just forwards the call to the superclass? 🤔

Reply:
yes, because those super methods are protected and we need to call them from here